training_mod_consts/src/options.rs

```diff
@@ -440,6 +440,13 @@ impl OnOff {
             _ => None,
         }
     }
+
+    pub fn as_bool(self) -> Option<bool> {
```

Comment:
> I'd probably avoid using From / Into for this case. Instead, I'd create a predicate method on the type. This will be more readable and can take &self, allowing you to continue using the value.

^ from linked implementation